haproxy_autoscale/haproxy_autoscale.py

```diff
@@ -7,6 +7,7 @@
 
 __version__ = '0.4.1'
```

Comment:
It looks like I missed this in the last PR, but could you update `__version__` and resubmit the PR?